### ISSUE TITLE
[Feature] Additional Clickable Text in Global Chat

### ISSUE THREAD

**Issue by Basically-Andrew** [EarthMC/Issue-Tracker#1374]
### Is your feature request related to a problem? Please describe.

Currently, players can click on a player's username in a chat message and quickly send a private message. This works by automatically filling a player's chat window with _/msg_ space the player's username. (i.e., _/msg Basically_Andrew_) I believe this is the only use of clickable chat messages, excluding links. 

There is no clickable for the town and nation prefix in front of a player's username (_[town|nation]_). Players can't click the mention of a town or nation in chat to see the _/town_ or _/nation_ information to consider joining or learn more. Players can't begin teleportation to a mentioned nation to shop or visit.

### Describe the solution you'd like

An optimal solution to the above issues would be adding the following clickable messages.
1. Click the town part of _[town|nation]_ to be shown the town's _/town_ page.
2. Click the nation part of _[town|nation]_ to be shown the nation's _/nation_ page.
3. Click the mention of a nation in a chat message to either fill the player's chat window with _/n spawn nationname_ or begin teleportation to the nation's capital.

Many other key chat messages could become clickable, but the above is what I could see new and old players using on earthmc. Also, you don't want to add too many clickable messages where clickable areas are overlapping and causing issues.

### Additional context

Below is the menu that currently appears when hovering over a player's username in chat.
<img src="https://user-images.githubusercontent.com/59465284/126050645-f7e58307-99e5-4074-83a4-6ae80f42d285.png" width="388" height="60">

**Comment by KarlOfDuty** [comment]
I agree with 1 and 2, not 3. We should not be parsing out nation names from chat messages.

**Comment by Basically-Andrew** [comment]
> I agree with 1 and 2, not 3. We should not be parsing out nation names from chat messages.

That's fair. But in my opinion, players would be more interested in clicking the nation part of _[town|nation]_ to fill the player's chat window with /n spawn nationname or begin teleportation than the nation's _/nation_ page. But that's my opinion. However, there is already a similar clickable on the _/n list_ page.

<img src="https://user-images.githubusercontent.com/59465284/126051103-6291402e-0693-4e9f-b73b-741363882c22.png" width="300" height="60">

**Comment by Basically-Andrew** [comment]
Also, after I voted I thought of another clickable, that being the announcement of players voting (Basically_Andrew has voted and received a gold crate! /vote). New and old players could quickly fill their chat window or instantly show the _/vote_ message/links. However, I'm unsure how feasible this is since /vote is an independent plugin if I'm correct.

**Comment by Warriorrrr** [comment]
I can do chat messages, but not the vote message since I don't know the source/name of the plugin.
